client/verta/verta/operations/monitoring/summaries.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+from builtins import object
```

Comment:
This isn't needed as far as I know—`object` is defined already in both Python 2 and 3.
```suggestion
```